core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/tasks/cni-plugins/canal.yml

```diff
@@ -1,4 +1,39 @@
 ---
+- name: Configure NetworkManager before attempting to use Canal networking
+  block:
+    # /etc/NetworkManager/conf.d directory is created by NetworkManager(rpm)/network-manager(deb) package
+    - name: Check if /etc/NetworkManager/conf.d exists
+      stat:
+        path: /etc/NetworkManager/conf.d
+        get_attributes: no
+      register: network_manager_conf_d_stat
+
+    # Remove flannel.1 interface from NetworkManager managed devices
+    - name: Copy flannel.conf to /etc/NetworkManager/conf.d
+      template:
+        src: network-manager.conf.j2
+        dest: /etc/NetworkManager/conf.d/flannel.conf
+        owner: root
+        group: root
+        mode: u=rw,g=r,o=r
+      register: copy_flannel_network_manager_conf
+      vars:
+        network_interface_prefix: flannel
+      when:
+        - network_manager_conf_d_stat.stat.exists
+
+- name: Reload NetworkManager service
+  when: copy_flannel_network_manager_conf.changed
+  block:
+    - name: Populate service facts
+      service_facts:
+
+    - name: Reload if NetworkManager service is running
+      systemd:
+        name: NetworkManager
+        state: reloaded
```

Comment:
Could you comment on why we need to reload NM? It's not obvious from the code itself (at least for me), and I believe this isn't always a safe operation.